docs/user-guide/release-notes/2.8.x.rst

```diff
@@ -2,6 +2,17 @@
 Pulp 2.8 Release Notes
 ======================
 
+Pulp 2.8.4
+==========
+
+Some of the improvements as resulf of :fixedbugs:`2.8.4` include:
+
+    * Errata installation on consumers occurs much faster. Yum and the security plugin on the
```

Comment:
I propose removing the sentence starting with `Yum and the`... I don't think this will mean anything to an end user.